shared/mad/codeql/mad/modelgenerator/internal/ModelGeneratorImpl.qll

```diff
@@ -605,6 +623,20 @@ module MakeModelGenerator<
       isField(ap.getAtIndex(_))
     }
 
+    /**
+     * Holds if this access path `ap` mentions a callback.
+     */
+    private predicate mentionsCallback(PropagateContentFlow::AccessPath ap) {
+      isCallback(ap.getAtIndex(_))
+    }
+
+    /**
+     * Models as Data currently doesn't support callback logic in fields.
```

Comment:
I don't think that's true: For example, I would expect something like this to work just fine:
```
input = Argument[0].Field[Foo].ReturnValue
output = ReturnValue
```